### DISCUSSION TITLE
How to set "Member privileges>Base permissions" as "Repository roles>Maintain"?

### DISCUSSION THREAD

**Discussion by ALIENQuake** [community/community#88691] [New to GitHub] [answered]
Hi,
How to set "Member privileges>Base permissions" as "Repository roles>Maintain"? 

The only one available are: Admin/Write/Read/No permission.

NOTE: I don't want to setup custom permissions.

Member privileges:
![image](https://github.com/community/community/assets/365255/313caf7a-3e73-4778-88fe-e01035b48d0f)

Repository roles:
![image](https://github.com/community/community/assets/365255/044c8ac7-bf94-427f-a9c2-c5cb3d5a13ef)

**Comment by AidenJamesYt** [comment, marked as answer]
Setting Member privileges -> Base permissions directly to Maintain isnt possible with standard GitHub settings. However, you can choose Write as the closest alternative, allowing members to manage most activities except sensitive or destructive actions. For finer control, adjust individual repository permissions for members/teams to Maintain. Keep in mind that base permissions apply globally, while specific repository permissions take precedence

**Reply by AidenJamesYt** [reply]
Did it work @ALIENQuake ?

**Reply by ALIENQuake** [reply]
@AidenJamesYt Yes but it's not ideal.

**Reply by AidenJamesYt** [reply]
Alright. Thats about all I can help with at this time, as thats all my knowledge about it

**Comment by ALIENQuake** [comment]
It works but it's not ideal. The "Maintain" represents ideal rights for my org: all members can accept PR (mostly cosmetic) for all repos and there is no need to manually handle new member's permissions. Does this looks like a good candidate for Feature Request?
